config/main.py

```diff
@@ -2187,6 +2194,68 @@ def override_config_by(golden_config_path):
     return
 
 
+def recover_hw_config(namespace_list=None):
```

Comment:
mac and platform is already supported. Can generate asic id here: https://github.com/sonic-net/sonic-utilities/pull/3711